### PR TITLE
Excercise 3-semantic-html

### PR DESCRIPTION
**Pull request by Jaimeandres97**

### CODE REVIEW

File: 3-semantic-html/semantic.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial=scale=1.0">
+        <meta name="description" content="Jaime Moreno Curriculum Vitae">
+        <meta name="keywords" content="CV, curriculum vitae, learning, job search">
```
Comment (juancho11gm) [inline 3-semantic-html/semantic.html:7]: Nice to see these meta tags.

File: 3-semantic-html/semantic.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                    <a href="https://www.worldometers.info/coronavirus/"> Coronavirus Stats</a>
+                </footer>
+            </article>
+        </main>
+
+        <aside>
+            <p class="aligncenter">
+                <img height="150" width="300" src="https://www.cdc.gov/coronavirus/2019-ncov/images/2019-coronavirus.png" alt="Coronavirus">
+            </p>
+            <style>.aligncenter{text-align:right;}</style>
```
Comment (juancho11gm) [inline 3-semantic-html/semantic.html:52]: Nice to see you using styles. We are going to see the difference between using `<style>` tag and using `<link rel="stylesheet" href="style.css">` inside the `<head>`.

File: 3-semantic-html/semantic.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                <nav>
+                    <p>You can contact the author here:</p>
+                        <ol>
+                            <li><a href="https://www.linkedin.com/in/jaimemoreno97/"> LinkedIn</a></li>
+                            <li><a href="https://www.facebook.com/jaimeandres97"> Facebook</a></li>
+
+                    </ol>
+                </nav>
+            </address>
+        </footer>
```
Comment (juancho11gm) [inline 3-semantic-html/semantic.html]: Please separate the `<address>` of the `<nav>`. You can have both of them inside the `<footer>`, but it's better if you separate them. Something like this: 
```html
<footer>
             <p>You can contact the author here:</p>
            <address>
                <a href="mailto:pushdev.cuentas@gmail.com">pushdev.cuentas@gmail.com</a><br>
                <a href="tel:+13115552368">(311) 555-2368</a>
            </address>
            <p>Content related:</p>
            <nav>
                    <!-- your list items -->
            </nav>
 </footer>
```

**Review by juancho11gm — CHANGES_REQUESTED** [review]
Good job, I see that you are using the tags that we saw in the session. Please see the comments and send the changes. You can add media type tags like <audio> or the others (if you want). Keep it up!

**Review by pushdev-code — APPROVED** [review]

**Pull request merged**